aiocache/decorators.py

```diff
@@ -205,7 +207,12 @@ async def decorator(self, f, *args, **kwargs):
 
 
 def _get_cache(cache=Cache.MEMORY, serializer=None, plugins=None, **cache_kwargs):
-    return Cache(cache, serializer=serializer, plugins=plugins, **cache_kwargs)
+    return Cache(
```

Comment:
Looks unrelated (maybe I need to tweak a Black config or something...).

Reply:
I've deleted space characters at ends of lines here.